distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -2416,6 +2514,58 @@ public void testAsyncDownloadActionPresentUsingCometDClient() throws Exception {
                 is(expectedUrl));
     }
 
+    protected List<Matcher<?>> getXpathMatchers(String expectedOutputSchema,
```

Comment:
What follows is a bunch of helper methods to cut down on duplicated code in the `@Test`methods.